google/resource_compute_ssl_policy.go

```diff
@@ -0,0 +1,281 @@
+package google
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	computeBeta "google.golang.org/api/compute/v0.beta"
+)
+
+func resourceComputeSslPolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeSslPolicyCreate,
+		Read:   resourceComputeSslPolicyRead,
+		Update: resourceComputeSslPolicyUpdate,
+		Delete: resourceComputeSslPolicyDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(2 * time.Minute),
+			Update: schema.DefaultTimeout(2 * time.Minute),
+			Delete: schema.DefaultTimeout(2 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+
+			"custom_features": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
this is surprising to me, can you confirm that the description can't be changed?

Reply:
I went off of the details provided in `gcloud beta compute ssl-policies update`, which doesn't state that description is an updatable field, but I'll try against the API directly and see if it can be patched.

Reply:
I just tried updating two fields via API Explorer: `minTlsVersion` and `description`. The change was applied to `minTlsVersion` but not `description`, so I believe the `ForceNew: true` is valid. If someone wants to change the resource description, a new resource must be created.